examples/ota-requestor-app/efr32/BUILD.gn

```diff
@@ -126,6 +126,8 @@ efr32_executable("ota_requestor_app") {
   if (show_qr_code) {
     sources += [ "${examples_plat_dir}/display/lcd.c" ]
     defines += [ "DISPLAY_ENABLED" ]
+
+    deps += [ "${chip_root}/examples/common/QRCode" ]
```

Comment:
Forgot to removed it from the deps above

Reply:
done